app/src/main/java/ipvc/gymbuddy/app/fragments/SettingsFragment.kt

```diff
@@ -1,8 +1,62 @@
 package ipvc.gymbuddy.app.fragments
 
+import android.os.Bundle
+import android.view.View
+import ipvc.gymbuddy.app.R
+import android.content.res.Configuration
+import java.util.Locale
 import ipvc.gymbuddy.app.core.BaseFragment
 import ipvc.gymbuddy.app.databinding.FragmentSettingsBinding
 
-class SettingsFragment : BaseFragment<FragmentSettingsBinding>(
-    FragmentSettingsBinding::inflate
-)
+class SettingsFragment : BaseFragment<FragmentSettingsBinding>(FragmentSettingsBinding::inflate) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.changeLanguage.setOnClickListener {
+            toggleLanguageOptions()
+        }
+
+        binding.englishOption.setOnClickListener { selectLanguage(it) }
+        binding.portugueseOption.setOnClickListener { selectLanguage(it) }
+    }
+
+    private fun toggleLanguageOptions() {
+        if (binding.languageOptions.visibility == View.GONE) {
+            binding.languageOptions.visibility = View.VISIBLE
+            binding.changeLanguage.setCompoundDrawablesWithIntrinsicBounds(R.drawable.globe, 0, R.drawable.baseline_arrow_drop_up_24, 0)
```

Comment:
does the arrow look exactly the same like the right arrow from the "Logout" button?